src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,42 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy");
+
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
+        CinemaHallService cinemaHallService = (CinemaHallService)
+                injector.getInstance(CinemaHallService.class);
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
+        LocalDate localDate = LocalDate.now();
+
+        Movie fastAndFurious = new Movie("Fast and Furious",
+                "An action film about street racing, heists, and spies.");
+        CinemaHall cinemaHall3D = new CinemaHall(15,
+                "3D screen and 8D sound");
+        MovieSession movieSession = new MovieSession(fastAndFurious, cinemaHall3D, localDate);
 
-        Movie fastAndFurious = new Movie("Fast and Furious");
-        fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
-        System.out.println(movieService.get(fastAndFurious.getId()));
-        movieService.getAll().forEach(System.out::println);
+        cinemaHallService.add(cinemaHall3D);
+        movieSessionService.add(movieSession);
+
+        System.out.println(movieService.get(1L));
```

Comment:
don't hardcode IDs, retrieve by getId(), fix it everywhere
```suggestion
        System.out.println(movieService.get(fastAndFurious.getId()));
```